uvicorn/protocols/utils.py

```diff
@@ -1,5 +1,5 @@
 import asyncio
-import urllib
+import urllib.parse
```

Comment:
why do we need that ?

Reply:
`urllib` doesn't import the module `parse`, and the only call we make on this file is here:

```python
    path_with_query_string = urllib.parse.quote(
        scope.get("root_path", "") + scope["path"]
    )
```
With the current `import urllib`, you can't travel to `quote()` due to the reason above.

But if you want a better excuse: it restrict the scope of what is being imported.

Reply:
ok quite unrelated to the current PR but that's minor ;)

Reply:
Oh, yeah, sorry. I don't know why I add it here :sweat_smile: